detections/application/cisco_secure_application_alerts.yml

```diff
@@ -0,0 +1,88 @@
+  * Providing visibility into attacker tactics, including source IP, attack techniques, and affected applications.
+  * Generating risk-based scoring and contextual alerts to enhance decision-making within SOC workflows.
+  * Helping analysts determine whether an attack was merely an attempt or if it successfully exploited a vulnerability.
+
+  By leveraging this information, SOC teams can proactively mitigate security threats, patch vulnerable applications, and enforce security controls to prevent further exploitation.
+data_source:
+- Cisco Secure Application AppDynamics Alerts
+search: |-
+  `appdynamics_security` blocked=false 
+  | rename attackEvents{}.attackOutcome AS attackOutcome, "attackEvents{}.vulnerabilityInfo.*" AS * 
+  | fields - tag::eventtype, eventtype, host, id, index, linecount, punct, source, sourcetype, splunk_server, tag, SourceType, app clientAddressType, application, tier, "attackEvents{}.*" 
+  | eval socketOut=mvjoin(socketOut," AND ") 
+  | eval risk_score=kennaScore 
+  | fillnull risk_score value="0" 
+  | eval risk_object=app_name 
+  | stats values(*) as * by attackId 
+  | eval severity=case(
+      risk_score>=100 OR signature="LOG4J", "critical",
+      risk_score>50 AND risk_score<75, "high",
+      risk_score=0 AND attackOutcome="EXPLOITED", "high",
+      risk_score<=50 AND attackOutcome!="OBSERVED", "medium",
+      risk_score=0 AND attackOutcome="ATTEMPTED", "medium",
+      risk_score=0, "low",
+      risk_score=0 AND attackOutcome="OBSERVED", "low"
+      ) 
+  | eval risk_message=case(
+      (signature="API" OR signature="LOG4J" OR signature="SSRF"), "An ".attackOutcome." ".signature." vulnerability is attempted to be abused from ".src_category." IP address ".src_ip." and was seen connecting to server ".dest_nt_host." hosting application ".app_name." and possibly exfiltrating data to ".socketOut."",
+      (signature="MALIP" OR signature="SQL"), "A vulnerability is being ".attackOutcome." to be abused from ".src_category." IP address ".src_ip." and was seen connecting to server ".dest_nt_host." hosting application ".app_name.".",
+      (signature="DESEREAL"), "The application ".app_name." deserializes untrusted data without sufficiently verifying that the resulting data will be valid. Data which is untrusted cannot be trusted to be well-formed. Malformed data or unexpected data could be used to abuse application logic, deny service, or execute arbitrary code, when deserialized."
+      )
+   | `cisco_secure_application_alerts_filter`
+how_to_implement: In order to properly run this search, you need to ingest alerts data from AppD SecureApp, specifically ingesting data via HEC. You will also need to ensure that the data is going to sourcetype appdynamics_security. You will need to install the Splunk Add-on for AppDynamics. This add-on will give the needed field aliases to properly run this search. In a future update you will be able to run this detection if ingesting data via the TA.
```

Comment:
The last sentence makes it sound like its impossible to use this detection with the TA today? Don't really think this is clear.

Reply:
i removed that part from how to implement!